Scripts/variable_dictionary.m

```diff
@@ -1,5 +1,20 @@
+%{
+This scripts provides a variable dictionary with values of control
+parameters, vehicle parameters and trailer parameters that are used in the
+simulation .
+%}
```

Comment:
Here to... %% please see [Code section spec Matlab](https://www.mathworks.com/help/matlab/matlab_prog/run-sections-of-programs.html)